tests/sentry/issues/test_escalating.py

```diff
@@ -246,11 +235,16 @@ def test_daily_count_query(self) -> None:
             )
             group = event.group
 
-        # The group had 2 events yesterday
-        one_day_ago_mins = 24 * 60
+        # The group had 2 events yesterday, 10 mins before midnight
+        # Tests that events are aggregated in the daily count query by date, not by 24 hr periods
+        now = datetime.now()
+        mins_since_ten_mins_before_today = (
+            int((now - now.replace(hour=0, minute=0, second=0, microsecond=0)).total_seconds() / 60)
+            + 10
```

Comment:
modified this test so that yesterday's events occurred 10 mins before today, instead of 24 hours ago